README.md

```diff
@@ -126,15 +139,24 @@ docker-compose run web database add-directory <DIRECTORY> [PREFIX]
 docker-compose run web database update-github-fields
 ```
 
-If you want to explore or edit database manually, you can connect database
-with `psql` command.
+If you want to explore or edit database manually, you can connect to the database
+with the `psql` command.
 
 ```sh
 # this will print the name of the container it starts
 docker-compose run -d db
 docker exec -it <the container name goes here> psql -U cratesfyi
 ```
 
+#### `daemon` subcommand
+
+```sh
+# Run a persistent daemon which queues builds and starts a web server.
+# Warning: This will try to queue hundreds of packages on crates.io, only start it
+if you have enough resources!
```

Comment:
Nit: missing comment.

Reply:
I'm not sure what you mean here, what comment should I add?

Reply:
Oh I'm missing the `#` I see

Reply:
Sorry, should've clarified a bit more!